contracts/core/src/libraries/ConfLevel.sol

```diff
@@ -5,24 +5,15 @@ pragma solidity ^0.8.12;
  * @title ConfLevel
  * @notice XMsg confirmation levels. Matches ConfLevels in lib/xchain/types.go
  * @dev We prefer explicit constants over Enums, because we want uint8 values to start at 1, not 0, as they do in
- *      lib/xchain/types.go, such that 0 can represent "unset".
+ *      lib/xchain/types.go, such that 0 can represent "unset". Note only latest and finalized levels are supported
+ *      on-chain.
  */
 library ConfLevel {
     /**
      * @notice XMsg confirmation level "latest", last byte of xmsg.shardId.
      */
     uint8 internal constant Latest = 1;
 
-    /**
-     * @notice XMsg confirmation level "fast", last byte of xmsg.shardId.
-     */
-    uint8 internal constant Fast = 2;
-
-    /**
-     * @notice XMsg confirmation level "safe", last byte of xmsg.shardId.
-     */
-    uint8 internal constant Safe = 3;
-
     /**
```

Comment:
latest and finalized to 1 & 2?

Reply:
They are still 1 & 4 in lib/xchain/types.go

Reply:
If we change them there I'm down